noise-maker.js

```diff
@@ -0,0 +1,44 @@
+const Synth= new Tone.Synth();
+Synth.toMaster();
+//Synth.triggerAttackRelease("C4","8n");
+
+const Green = document.getElementById("green");
+green.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Blue = document.getElementById("blue");
+blue.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Red = document.getElementById("red");
+red.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
+});
+const Yellow = document.getElementById("yellow");
+yellow.addEventListener("mousedown",e=>{
+  const note = e.target.dataset.note
+  Synth.triggerAttackRelease(note,"8n")
```

Comment:
Also think reducing this instance by using a function. Can't think up the details real quick. I did something similar. And will make that change too.